soc/riscv/riscv-ite/it8xxx2/Kconfig.defconfig.series

```diff
@@ -16,19 +16,57 @@ config ITE_IT8XXX2_INTC
 config RISCV_GP
 	default y
 
+config ITE_IT8XXX2_TIMER
+	default y
+
 config SYS_CLOCK_HW_CYCLES_PER_SEC
 	default 32768
 
 config SYS_CLOCK_TICKS_PER_SEC
 	default 32768
 
+config UART_NS16550
+	default y
+	depends on SERIAL
+
 config UART_NS16550_WA_ISR_REENABLE_INTERRUPT
 	default y
 	depends on UART_NS16550
 
 config RISCV_HAS_CPU_IDLE
 	default y
 
+config PINMUX_ITE_IT8XXX2
+	default y
+	depends on PINMUX
+
+config WDT_ITE_IT8XXX2
+	default y
+	depends on WATCHDOG
+
+config KSCAN_ITE_IT8XXX2
+	default y
+	depends on KSCAN
+
+config GPIO_ITE_IT8XXX2
+	default y
+	depends on GPIO
+
+config I2C_ITE_IT8XXX2
+	default y
+	depends on I2C
+
+config ADC_ITE_IT8XXX2
+	default y
+	depends on ADC
+
+config SOC_FLASH_ITE_IT8XXX2
+	default y
```

Comment:
`depends on FLASH`

Reply:
Done

Reply:
So if I want to build the it8xxx2_evb with tachometer driver, then I should add the CONFIG_SENSOR=y in samples/boards/it8xxx2_evb/prj.conf (if the path exists) , right?

Reply:
Yes